.gitignore

```diff
@@ -111,3 +111,7 @@ atlassian-ide-plugin.xml
 
 # Ignore local configuration files for asdf, allowing the JDK to be configured for project (https://asdf-vm.com)
 .tool-versions
+
+# gradle-profiler
+gradle-user-home
+profile-out*
```

Comment:
🤔 In general, it isn't a good idea to put the gradle user home or the profile-out in the project directory, since it creates additional file system events that may change the performance of the build. It is better to have both directories outside of the project directory. So I am not sure it is a good idea to add these ignores since they might encourage folks to put the directories in the project directory.

Reply:
These are where Gradle profiler puts these files automatically. 

Should we change the defaults of Gradle profiler?

Or perhaps Gradle profiler will let me set the project directory to another dir than cwd.

it does seem easy to shoot yourself in the foot with the current behavior

Reply:
> Or perhaps Gradle profiler will let me set the project directory to another dir than cwd.

Yes, that is what you should be doing. I agree it isn't a great experience currently to discover this.

Reply:
This sounds like something we should enforce in Gradle profiler itself. I can imagine I’m not the only one to run into this

Reply:
The profiler should fail if cwd == project directory